lighthouse-core/config/config.js

```diff
@@ -45,6 +66,17 @@ function assertValidPasses(passes, audits) {
     });
   });
 
+  // All required gatherers must be found in the config. Throw otherwise.
+  for (const auditDefn of audits || []) {
+    const auditMeta = auditDefn.implementation.meta;
+    for (const requiredArtifact of auditMeta.requiredArtifacts) {
+      if (!foundGatherers.has(requiredArtifact)) {
+        throw new Error(`${requiredArtifact} gatherer, required by audit ${auditMeta.id}, ` +
```

Comment:
should we make this an i18n'd LHError too while we're adding it? I'd personally vote no on all config errors since we never expect these to happen in blessed environments :)

Reply:
> should we make this an i18n'd LHError too while we're adding it? I'd personally vote no on all config errors since we never expect these to happen in blessed environments :)

yeah, I'll vote no too. Luckily like a million tests will fail if the default config starts throwing this error :)